Mobile Buy SDK/Mobile Buy SDK/Models/BUYImage.m

```diff
@@ -43,4 +43,22 @@ - (void)updateWithDictionary:(NSDictionary *)dictionary
 	_updatedAtDate = [dateFormatter dateFromString:dictionary[@"updated_at"]];
 }
 
+#pragma mark - BUYPersistence
+
+- (NSDictionary *)plistDictionary
+{
+	NSMutableDictionary *dictionary = [[super plistDictionary] mutableCopy];
+	
+	if (self.src)           dictionary[@"src"] = self.src;
```

Comment:
All `if` (and `else`) statement bodies must be wrapped in curly braces, and on separate lines, for safe coding reasons.